### PR TITLE
[DOCS-7420] Enable Advanced Search in Attribute Search

### PR DESCRIPTION
**Pull request by Prosune**
Ignore DOCS-7124. Please review DOCS-7420

### CODE REVIEW

File: content-accelerator/latest/using/user-guide.md [hunk 1]
```diff
@@ -1154,6 +1155,65 @@ you to search for all possible documents in the repository.
...
+
+    ![Advanced Search additional configuration]({% link content-accelerator/images/aca-show-advanced-search-options-config.png %})
+
+When the Enable Any/All Search setting is enabled, you can configure either of the following settings.
+
+| Settings | Search operator |
+| -------- | --------------- |
+| ANY of these values | If you select this option, search is performed based on the OR operator |
+| ALL of these values |	If you select this option, search is performed based on the AND operator |
+
```
Comment (dpatel-alfresco) [inline content-accelerator/latest/using/user-guide.md:1180]: Please add.

If the **Enable Any/All disabled** setting is disabled, the default operation is the AND operation.

File: content-accelerator/latest/using/user-guide.md [hunk 2]
```diff
@@ -1154,6 +1155,65 @@ you to search for all possible documents in the repository.
...
+
+| Object | Default operation |
+| ------ | ----------------- |
+| **AutoComplete** and **DropDown** | Exact search with case sensitivity |
+| **RadioButton** and **Cascading** | Exact search |
+| **TextBox** | The search depends on values set using “Make Search Exact” and “Case Sensitive Search” and if these values are not set, a wildcard search is performed. |
+
+**Notes:**
+
```
Comment (dpatel-alfresco) [inline content-accelerator/latest/using/user-guide.md:1210]: please mention it is available as on ACA 3.6.x .

**Review by dpatel-alfresco — CHANGES_REQUESTED** [review]
Please add suggested changes

File: content-accelerator/latest/using/user-guide.md (hunk 2 above)
Comment (dpatel-alfresco) [reply content-accelerator/latest/using/user-guide.md:1210]: Advanced Search is deprecated in Content Accelerator 3.6. Though it is still available, it is expected to be removed in future releases and replaced with Advanced Search capabilities added to Attribute Search, described above.

**Review by dpatel-alfresco — APPROVED** [review]
Looks good to me

**Review by anxumalo — APPROVED** [review]
LGTM

**Pull request merged**